content/docs/guides/flyway.md

```diff
@@ -0,0 +1,173 @@
+enableTableOfContents: true
+---
+
+Flyway is a database migration tool that facilitates version control for databases. It allows developers to manage and track changes to the database schema, ensuring that the database evolves consistently across different environments.
+
+This guide steps you through installing the Flyway command-line tool, configuring Flyway to connect to a Neon database, and running database migrations. The guide follows the setup described in the [Flyway command-line quickstart](https://documentation.red-gate.com/fd/quickstart-command-line-184127576.html).
+
+## Prerequisites
+
+- A Neon account. See [Sign up](/docs/get-started-with-neon/signing-up).
+- A Neon project. See [Create your first project](/docs/get-started-with-neon/setting-up-a-project).
+- A database. This guide uses the ready-to-use `neondb` database. You can create your own database if you like. See [Create a database](/docs/manage/databases#create-a-database) for instructions.
+
+## Download and extract Flyway
+
+1. Download the [Flyway command-line tool](https://documentation.red-gate.com/fd/command-line-184127404.html).
+
+2. Extract the Flyway files. For example:
+
+    ```bash
+    cd ~/Downloads
+    tar -xzvf flyway-commandline-9.22.3-linux-x64.tar.gz -C ~/
+    ```
+
+3. Open a command prompt to view the contents of your Flyway installation:
+
+    ```bash
+    cd ~/flyway-9.22.3
+    ls
+    assets  drivers  flyway.cmd  jre  licenses    rules
+    conf    flyway   jars        lib  README.txt  sql
+    ```
+
+## Set your path variable
+
+Add the Flyway directory to your `PATH` so that you can execute Flyway commands from any location. For example:
+
+```bash
+echo 'export PATH=$PATH:~/flyway-9.22.3' >> ~/.bashrc
+source ~/.bashrc
+```
+
+or
+
+```bash
+echo 'export PATH=$PATH:~/flyway-9.22.3' >> ~/.profile
+source ~/.profile
+```
+
+## Retrieve your Neon database connection string
+
+From the Neon **Dashboard**, retrieve your password and a Java connection string from the **Connection Details** widget.
+
+Your Java connection string should look something like this:
+
+<CodeBlock shouldWrap>
+
+```bash
+jdbc:postgresql://ep-cool-darkness-123456.us-east-2.aws.neon.tech/neondb?user=alex&password=AbC123dEf
+```
+
+</CodeBlock>
+
+## Configure flyway
+
+Configure Flyway to connect to your Neon database by uncommenting and updating the following items in the `/conf/flyway.conf` file with the connection details you retrieved in the previous step.
+
+<CodeBlock shouldWrap>
+
+```bash
+flyway.url=jdbc:postgresql://ep-cool-darkness-123456.us-east-2.aws.neon.tech:5432/neondb
+
+flyway.user=alex
+
+flyway.password=AbC123dEf
+
+flyway.locations=filesystem:/home/alex/flyway-9.22.3/sql
+```
+
+</CodeBlock>
+
+## Create the first migration
+
+Create your first migration in the `/sql` directory. This migration creates a `person` table in your database. We'll name the migration file `V1__Create_person_table.sql`:
```

Comment:
For me, the /sql directory was not created by default. It sounds like, by convention, it is usually created at the same level as `/conf`.  Consider:

"Create an `sql` directory to hold your first migration file. We'll name the file `V1__Create_person_table.sql` and include the following command, which creates a `person` table in your database."

Reply:
fixed.